cppcache/src/ThinClientRedundancyManager.hpp

```diff
@@ -26,6 +26,8 @@
 #include <set>
 #include <string>
 
+#include <ace/ACE.h>
```

Comment:
NNnnnoooooooooooo!  Actually think this is vestigial from some earlier change you made or something.  We removed it and built successfully.